js/constants/appConfig.js

```diff
@@ -36,8 +36,8 @@ module.exports = {
   cookieblock: {
     enabled: true
   },
-  autoplay: {
-    enabled: false
+  noAutoplay: {
+    enabled: true
```

Comment:
Per our discussions, we want to default this to false, right?  So that content isn't blocked
cc: @bradleyrichter